docs/docs/coverage/language/java.md

```diff
@@ -12,12 +12,12 @@ Each artifact supports the following scanners:
 
 The following table provides an outline of the features Trivy offers.
 
-| Artifact         |    Internet access    | Dev dependencies | [Dependency graph][dependency-graph] | Position | [Detection Priority][detection-priority] |
-|------------------|:---------------------:|:----------------:|:------------------------------------:|:--------:|:----------------------------------------:|
-| JAR/WAR/PAR/EAR  |     Trivy Java DB     |     Include      |                  -                   |    -     |                Not needed                |
-| pom.xml          | Maven repository [^1] |     Exclude      |                  ✓                   |  ✓[^7]   |                    -                     |
-| *gradle.lockfile |           -           |     Exclude      |                  ✓                   |    ✓     |                Not needed                |
-| *.sbt.lock       |           -           |     Exclude      |                  -                   |    ✓     |                Not needed                |
+| Artifact         |    Internet access    |  Dev dependencies  | [Dependency graph][dependency-graph] | Position | [Detection Priority][detection-priority] |
+|------------------|:---------------------:|:------------------:|:------------------------------------:|:--------:|:----------------------------------------:|
+| JAR/WAR/PAR/EAR  |     Trivy Java DB     |      Include       |                  -                   |    -     |                Not needed                |
+| pom.xml          | Maven repository [^1] | [Include](#scopes) |                  ✓                   |  ✓[^7]   |                    -                     |
```

Comment:
You changed from "Exclude" to "Include", but IMO, it should describe the default behavior. Actually, Node.js says "Excluded" although it supports `--include-dev-deps`. If you think "Include" is better, we need to change Node.js and other ecosystems for consistency.

```suggestion
| pom.xml          | Maven repository [^1] | [Exclude](#scopes) |                  ✓                   |  ✓[^7]   |                    -                     |
```

Reply:
Hmm… 
This is a very controversial situation, but I would probably agree with you that we should specify the default behavior in this table.
Updated in https://github.com/aquasecurity/trivy/pull/7414/commits/117555f15ec0d6daacff5dfdcd2f83b368364658